src/StoreKeeper/WooCommerce/B2C/Frontend/Handlers/Seo.php

```diff
@@ -2,105 +2,35 @@
+    public function prepareSeo($markdown, $product)
+    {
+        $selectedHandler = StoreKeeperOptions::get(StoreKeeperOptions::SEO_HANDLER, self::STOREKEEPER_HANDLER);
 
-            // get minimum price instead of first variation
-            if ($product instanceof \WC_Product_Variable) {
-                $offer['price'] = $product->get_variation_price('min', false);
-            }
+        if (self::STOREKEEPER_HANDLER === $selectedHandler) {
+            $handler = new StorekeeperHandler();
+            $handler->handle($markdown, $product);
         }
-
-        return $markdown;
     }
 
-    /**
-     * @param $markdown
-     * @param $product
-     * @param array $featured_attrs
-     */
-    protected function applyBarcode(&$markdown, $product)
+    public static function isYoastActive(): bool
     {
-        $barcode_name = FeaturedAttributeOptions::getWooCommerceAttributeName(FeaturedAttributes::ALIAS_BARCODE);
-        if (!empty($barcode_name)) {
-            $barcode = $product->get_attribute($barcode_name);
-            if (!empty($barcode)) {
-                $matched = false;
-                if (preg_match('/^[\d-\s]{8,}$/', $barcode)) {
-                    // only numbers (space and minus)
-                    $barcode = preg_replace('/[-\s]/', '', $barcode);
-                    if (8 === strlen($barcode)) {
-                        $markdown['gtin8'] = $barcode;
-                        $matched = true;
-                    } else {
-                        if (12 === strlen($barcode)) {
-                            $markdown['gtin12'] = $barcode;
-                            $matched = true;
-                        } else {
-                            if (14 === strlen($barcode)) {
-                                $markdown['gtin14'] = $barcode;
-                                $matched = true;
-                            } else {
-                                if (13 === strlen($barcode)) {
-                                    // check prefixes for ISBN
-                                    // see: https://en.wikipedia.org/wiki/International_Standard_Book_Number#Overview
-                                    $first3 = substr($barcode, 0, 3);
-                                    if ('978' === $first3 || '979' === $first3) {
-                                        $markdown['isbn'] = $barcode;
-                                        $matched = true;
-                                    } else {
-                                        $markdown['gtin13'] = $barcode;
-                                        $matched = true;
-                                    }
-                                }
-                            }
-                        }
-                    }
-                }
-                if (!$matched) {
-                    $markdown['mpn'] = $barcode;
-                }
-            }
-        }
-    }
+        $activePlugins = apply_filters('active_plugins', get_option('active_plugins'));
```

Comment:
we already have a class for checking if plugins are active `src/StoreKeeper/WooCommerce/B2C/Objects/PluginStatus.php` add there cos all in one place

Reply:
Moved checking of yoast to `PluginStatus`.